docs/spec/broker.md

```diff
@@ -44,6 +44,14 @@ The attributes filter specifying a list of key-value pairs MUST be supported by
 Trigger. Events that pass the attributes filter MUST include context or
 extension attributes that match all key-value pairs exactly.
 
+### Delivery Spec
+
+Both BrokerSpec and TriggerSpec have a `Delivery` field of type `duck.DeliverySpec` (per `v1`).
+This field allows the user to define the dead letter sink and retries.
```

Comment:
Should we provide a link to the whole spec https://github.com/knative/eventing/blob/master/docs/spec/spec.md#eventingduckv1deliveryspec 
instead of listing it here. My concern is that if later we add anything to the DeliverySpec, this statement needs to be changed too. Or at least do sth like `such as defining the dead letter sink and retries`.

Reply:
Do we still maintain that list? I thought it was abandoned, but i see it was recently reworked https://github.com/knative/eventing/pull/4594
I can link, but i still prefer to have this sentence, just to give a quick overview of what this thing does

Reply:
I believe all the spec should be maintained after all the rework, so a link will make sense. I guess we can argue `allows` does not mean `all it can do`, but what I was suggesting is to make it less like "this is the full list it can do".

Reply:
Linked and modified a bit that sentence. How does it look like now?

Reply:
Perfect. Thanks.